### PR TITLE
Re-enable dual-uploads.

### PR DESCRIPTION
**Pull request by mattmoor**
This is a draft until we push to production to allow-list the identity.

### CODE REVIEW

**Review by k4leung4 — APPROVED** [review]

**Comment by k4leung4** [comment]
allow list change is now in prod

**Comment by mattmoor** [comment]
@k4leung4 I'll merge tomorrow when approvers are more readily available to rollback 😅   thanks for pushing!

**Pull request merged**
